fuel_test/localrc.default

```diff
@@ -4,6 +4,7 @@ export FM_NAME="Fuel8"
 export FM_SNAPSHOT="fuel_ci"
 export FM_MNT=""
 export FM_XVA="fuel8.xva"
+export FM_PWD="r00tme"
```

Comment:
We can't have the password here either :)
It'll either have to be an actual argument to the script (rather than an env var) or passed on the command line, like FM_PWD="r00tme" fuel_test/prep_env.sh